WARN on char** for strtod

defining:
#define NK_STRTOD strtod
I got this WARN:
nuklear.h:6841:35: warning: passing argument 2 of ‘strtod’ from incompatible pointer type
note: expected ‘char ** restrict’ but argument is of type ‘const char **’

Casting in nk_strtof() line 6841 solve this WARN:
double_value = NK_STRTOD(str, (char **__restrict)endptr);

gcc version 12.2.0 (Debian 12.2.0-14)
libc6 ver: 2.36-9+deb12u7

--- a/src/nuklear.h
+++ b/src/nuklear.h
@@ -3612,11 +3612,11 @@ NK_API struct nk_vec2 nk_rect_size(struct nk_rect);
 NK_API int nk_strlen(const char *str);
 NK_API int nk_stricmp(const char *s1, const char *s2);
 NK_API int nk_stricmpn(const char *s1, const char *s2, int n);
-NK_API int nk_strtoi(const char *str, const char **endptr);
-NK_API float nk_strtof(const char *str, const char **endptr);
+NK_API int nk_strtoi(const char *str, char **endptr);
+NK_API float nk_strtof(const char *str, char **endptr);
 #ifndef NK_STRTOD
 #define NK_STRTOD nk_strtod
-NK_API double nk_strtod(const char *str, const char **endptr);
+NK_API double nk_strtod(const char *str, char **endptr);
 #endif
 NK_API int nk_strfilter(const char *text, const char *regexp);
 NK_API int nk_strmatch_fuzzy_string(char const *str, char const *pattern, int *out_score);

--- a/src/nuklear_util.c
+++ b/src/nuklear_util.c
@@ -141,7 +141,7 @@ nk_strlen(const char *str)
     return siz;
 }
 NK_API int
-nk_strtoi(const char *str, const char **endptr)
+nk_strtoi(const char *str, char **endptr)
 {
     int neg = 1;
     const char *p = str;
@@ -165,7 +165,7 @@ nk_strtoi(const char *str, const char **endptr)
     return neg*value;
 }
 NK_API double
-nk_strtod(const char *str, const char **endptr)
+nk_strtod(const char *str, char **endptr)
 {
     double m;
     double neg = 1.0;
@@ -222,7 +222,7 @@ nk_strtod(const char *str, const char **endptr)
     return number;
 }
 NK_API float
-nk_strtof(const char *str, const char **endptr)
+nk_strtof(const char *str, char **endptr)
 {
     float float_value;
     double double_value;

